packages/datadog-plugin-koa/test/suite.js

```diff
@@ -0,0 +1,6 @@
+'use strict'
+const suiteTest = require('../../dd-trace/test/plugins/suite')
+
+suiteTest('koa', 'koajs/koa', 'latest')
+
+// suiteTest('@koa/router', 'koajs/router', 'latest')
```

Comment:
Was this commented out by accident? Otherwise it should be removed or a comment should be added to explain.

Reply:
I had some issues running it. I'll add a TODO to re-enable it.